app/code/Magebit/Faq/Model/Question.php

```diff
@@ -0,0 +1,228 @@
+        AbstractDb $resourceCollection = null,
+        array $data = [],
+        ?WYSIWYGValidatorInterface $wysiwygValidator = null,
+        CompositeUrlKey $compositeUrlValidator = null
+    ) {
+        parent::__construct($context, $registry, $resource, $resourceCollection, $data);
+        $this->wysiwygValidator = $wysiwygValidator
+            ?? ObjectManager::getInstance()->get(WYSIWYGValidatorInterface::class);
+        $this->compositeUrlValidator = $compositeUrlValidator
+            ?? ObjectManager::getInstance()->get(CompositeUrlKey::class);
+    }
+
+    /**
+     * Construct.
+     *
+     * @return void
+     */
+    protected function _construct()
+    {
+        $this->_init(\Magebit\Faq\Model\ResourceModel\Question::class);
+    }
+
+    /**
+     * Retrieve question id
+     *
+     * @return string
+     */
+    public function getId(): ?string
+    {
+        return $this->getData(self::ID);
+    }
+
+    /**
+     * Retrive question title
+     *
+     * @return string
+     */
+    public function getQuestion(): string
+    {
+        return $this->getData(self::QUESTION);
+    }
+
+    /**
+     * Retrive question's answer
+     *
+     * @return string
+     */
+    public function getAnswer(): string
+    {
+        return $this->getData(self::ANSWER);
+    }
+
+    /**
+     * Retrive question status
+     *
+     * @return string
+     */
+    public function getStatus(): string
+    {
+        return $this->getData(self::STATUS);
+    }
+
+    /**
+     * Retrive question position
+     *
+     * @return int
+     */
+    public function getPosition(): int
+    {
+        return $this->getData(self::POSITION);
+    }
+
+    /**
+     * Retrive question last modified time
+     *
+     * @return string
+     */
+    public function getUpdateAt(): string
+    {
+        return $this->getData(self::UPDATED_AT);
+    }
+
+    /**
+     * Set title
+     *
+     * @param string $question
+     * @return QuestionInterface
+     */
+    public function setQuestion($question): QuestionInterface
+    {
+        return $this->setData(self::QUESTION, $question);
+    }
+
+    /**
+     * Set answer
+     *
+     * @param string $answer
+     * @return QuestionInterface
+     */
+    public function setAnswer($answer): QuestionInterface
+    {
+        return $this->setData(self::ANSWER, $answer);
+    }
+
+    /**
+     * Set status
+     *
+     * @param int $status
+     * @return QuestionInterface
+     */
+    public function setStatus($status): QuestionInterface
+    {
+        return $this->setData(self::STATUS, $status);
+    }
+
+    /**
+     * Set position
+     *
+     * @param int $position
+     * @return QuestionInterface
+     */
+    public function setPosition($position): QuestionInterface
+    {
+        return $this->setData(self::POSITION, $position);
+    }
+
+    /**
+     * Prepare question's statuses.
+     *
+     * @return array
+     */
+    public function getAvailableStatuses()
```

Comment:
Model entity should contain setter and getter methods for:

- id (only getter)
- question
- answer
- status
- position
- updated_at (only getter)

- The newly created code follows the best practices of Magento development and doesn't contain any junk files or code.